runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerConfig.java

```diff
@@ -32,10 +32,33 @@
 @ConfigGroup("loottracker")
 public interface LootTrackerConfig extends Config
 {
+	@ConfigItem(
+		keyName = "persistentData",
+		name = "Persistent data",
+		description = "Configures whether loot tracker data should persist between sessions.<br/> Must be Logged in.",
+		position = 0
+	)
+	default boolean persistentData()
+	{
+		return false;
+	}
+
+	@ConfigItem(
```

Comment:
This should be enabled by default, and it should mention that it requires login in name (e.g Historical data (Requires login!)